src/chrome/content/rules/LinuxFoundation.xml

```diff
@@ -40,100 +33,39 @@
 	<target host="lists.linux-foundation.org" />
```

Comment:
Same for `linuxfoundation.org`, but therefore this can be done in a follow up PR.